src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/StegService.kt

```diff
@@ -54,6 +54,14 @@ class StegService(
         return håndterSteg(saksbehandling, behandlingSteg, vedtak)
     }
 
+    @Transactional
+    fun håndterFerdigstilleVedtakUtenBeslutter(saksbehandling: Saksbehandling): Behandling {
+        håndterSendTilBeslutter(saksbehandling)
+        val oppdatertBehandling = behandlingService.hentSaksbehandling(saksbehandling.id)
+        val godkjentBesluttetVedtak = BeslutteVedtakDto(godkjent = true)
+        return håndterBeslutteVedtak(oppdatertBehandling, godkjentBesluttetVedtak)
```

Comment:
Finnes det noen risiko at vi ikke gjør de valideringer som gjøres i "SendTilBeslutter"?

Reply:
Nei, denne trigges egentlig av samme kode som `SendTilBeslutter`.  
Så utfordringen vil isåfall ligge hvis det kommer mer validering i Controller-funksjonen til `BeslutteSteg` - for den går aldri saksbehandleren innom. 

Per nå ligger det en sjekk på at underkjent totrinns må ha beskrivende tekst, men det er ikke relevant for "automatisk" godkjenning...